### PR TITLE
[FEAT] Adding valentines SFTs

### PR DESCRIPTION
**Pull request by HBandesh**
# Description

In this PR, I have added sfts for valentine's event. The following items have been added:
- Heart Balloons - 10 Love Letters. 100,000 supply
- Flamingo - 50 Love Letters. 10,000 supply. 
- Blossom Tree - 300 Love Letters. 1,000 Supply

image

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: metadata/metadata.ts [hunk 1]
```diff
@@ -3114,6 +3114,36 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
     image_url: "../public/erc1155/images/450.gif",
     attributes: [],
   },
+  "Heart Balloons": {
```
[the remark this answers is not in the document]
Comment (HBandesh) [reply metadata/metadata.ts:3117]: Thank you. Just did. :)

**Comment by Polysemouse** [comment]
The 3 new NFTs does not have the correct pixel scale in the map.  Perhaps they need to be fixed.

**Comment by Polysemouse** [comment]
@HBandesh The 3 new NFTs does not have the correct pixel scale in the map. Perhaps they need to be fixed.

File: src/features/island/collectibles/components/HeartBalloons.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import heartBalloons from "assets/events/valentine/sfts/balloons.webp";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+
+export const HeartBalloons: React.FC = () => {
+  return (
+    <>
+      <img
+        src={heartBalloons}
+        style={{
+          width: `${PIXEL_SCALE * 34}px`,
```
Comment (adamhannigan) [inline src/features/island/collectibles/components/HeartBalloons.tsx]: Screen Shot 2023-02-08 at 4 56 54 pm

`18` is the desired width 😄

**Review by adamhannigan — APPROVED** [review]
Great work mate! Love the descriptions ❤️ 

I've pushed a few sizing fixes to the assets and added some comments 😄

**Pull request merged**